packages/manager/apps/dedicated/client/app/ip/components/list/firewall/ip-ip-firewall.html

```diff
@@ -37,10 +37,14 @@ <h4 data-translate="ip_firewall_title"></h4>
         dismissable
         class="mt-2 mb-2"
     >
-        <span
-            data-translate="ip_firewall_deny"
-            data-translate-values="{ 'denyUrl': IpFirewallCtrl.denyUrl}"
-        ></span>
+        <span data-translate="ip_firewall_deny_part_1"></span>
+        <a
+            href="javascript:void(0)"
+            data-ng-click="IpFirewallCtrl.createDenyRule()"
+        >
+            <span data-translate="ip_firewall_deny_link"></span>
+        </a>
```

Comment:
```suggestion
        <a
            href="#"
            data-ng-click="IpFirewallCtrl.createDenyRule()"
        >
            <span data-translate="ip_firewall_deny_link"></span>
        </a>
```

Reply:
Nope, if I do your suggestion, the page will be reloaded and it's not what I want to do. I just want to have a link which  send to an action "createDenyRule".